### ISSUE TITLE
Cannot change active database

### ISSUE THREAD

**Issue by denizkaanaraci** [dbeaver/dbeaver#3810]
After updating 5.1.3 cannot change active database on postgresql. I wrote an issue about multi db support and you fixed it. But now i cannot change active database.

**Comment by denizkaanaraci** [comment]
Fixed problem is [#3687](https://github.com/dbeaver/dbeaver/issues/3687)

**Comment by serge-rider** [comment]
Can't reproduce. 
Perhaps this has the same nature as #3801.
Could you check latest EA version (https://dbeaver.io/files/ea/) - it contains fix with PostgreSQL databases switcher.

If it won't help please describe the problem a bit more. Debug/error logs would be very helpful.

**Comment by jyinpci** [comment]
In 5.1.2, with postgresql, if I double click on a database, it would become the active database automatically.  But with 5.1.3 and 5.1.4, I have to manually right click and set it to active database.  In fact, in the Edit Connection page, there is no checkbox for auto-switch anymore.  Is this the intended behaviour?  I very much prefer that in 5.1.2.

**Comment by serge-rider** [comment]
Yes, it is the standard behavior for all databases. Now PG works as all other databases.
But maybe the auto-switch behavior should be configurable. You could create a separate ticket for that.

**Comment by xgboosted** [comment]
I am facing the same issue in Azure SQL Server. Any help, please?

![image](https://user-images.githubusercontent.com/1528712/136874296-48355c63-74bc-4fc0-bee0-fe6fb1836858.png)
